CHANGES.md

```diff
@@ -1,3 +1,17 @@
+## 1.3.5 2018-01-17
+
+* H2 router and `io.l5d.mesh` Namerd interface are no longer experimental ([#1782](https://github.com/linkerd/linkerd/pull/1782)).
```

Comment:
Maybe silly but this is exciting enough to potentially use an "emoji"

Reply:
I will "add" an "emoji"